### ISSUE TITLE
Extract ontology metadata

### ISSUE THREAD

**Issue by dgarijo** [KnowledgeCaptureAndDiscovery/somef#531]
Right now only the URI of the ontology is returned. It would be great to have a proper ontology object retrieved (to be expanded in the spec)

**Comment by juanjemdIos** [comment]
I’ve just opened a PR, but I have a couple of questions. It's much more complete now because previously it was just a URL, and now it's a more detailed object. However, I’m not sure about the authors array. Is it fine as is since that's what gets extracted, or should we process it the same way we do for authors in other categories?

This is from widoco before improvement:
    "ontologies": [
        {
            "result": {
                "value": "https://raw.githubusercontent.com/dgarijo/Widoco/master/doc/benchmark/IFC4_ADD1.ttl",
                "type": "Url"

And now.....

    "ontologies": [
        {
            "result": {
                "value": "http://www.buildingsmart-tech.org/ifcOWL/IFC4_ADD1",
                "type": "Ontology",
                "url": "https://raw.githubusercontent.com/dgarijo/Widoco/master/doc/benchmark/IFC4_ADD1.ttl",
                "title": "IFC4_ADD1",
                "description": "OWL ontology for the IFC conceptual data schema and exchange file format for Building Information Model (BIM) data",
                "preferred_namespace_prefix": "ifc",
                "preferred_namespace_uri": "http://www.buildingsmart-tech.org/ifcOWL/IFC4_ADD1"
            },
            "confidence": 1,
            "technique": [
                "file_exploration"
            ]
        },

But look at my dude about authors. Some examples:

              "author": [
                    "http://www.mendeley.com/profiles/janne-parkkila/"
                ],
...
                "author": [
                    "http://businessgraph.io",
                    "http://ontotext.com",
                    "mailto:vladimir.alexiev@ontotext.com"
                ],
....
                "author": [
                    "Luis Manuel Vilches-Blazquez"
                ],
....
                "author": [
                    "http://earth.usc.edu/~khider/",
                    "http://www.cefns.nau.edu/~npm4/",
                    "http://orcid.org/0000-0001-5920-4751"
                ],

**Comment by dgarijo** [comment]
Yes, this opens a little of a can of worms. Authors may have their own URI and metadata. Expecting the range to be a string with the name is not correct. If the value is a URI, you need to query the ontology for that author metadata
